README.md

```diff
@@ -16,9 +16,136 @@ An engine for draw a timeline graph:
 
 ![](result-demo.jpg)
 
+# Install
+
+NPM
+
+```
+npm install short-night
+```
+
+# Usage
```

Comment:
In `README.md`, we should explain how to install and start development for short-night. Such build and setup related things. 

This section is more like a development manual, we should put them on our Wiki pages.

Reply:
Done.